libtalek/client_test.go

```diff
@@ -126,3 +126,42 @@ func TestRead(t *testing.T) {
 		t.Fatalf("Read wasn't for the enqueued subscription. %v / %v / %d", rv1, rv2, bucket)
 	}
 }
+
+func TestFragment(t *testing.T) {
+	theMsg := make([]byte, 1024*256)
+	validateMsg := make([]byte, 1024*256)
+
+	hdr := msgHeader{}
+	hdr.left = uint32(len(theMsg))
+
+	parts := make([][]byte, 0)
+	for hdr.left > 0 {
```

Comment:
uh, i don't think i'm gonna worry about that until we make a protocol where it's possible to happen.  right now it can skip ahead to the next full message if it desynchronizes, which seems like enough.